overrides/default.nix

```diff
@@ -2060,6 +2074,11 @@ lib.composeManyExtensions [
         '';
       });
 
+
+      pythonping = super.pythonping.overridePythonAttrs (old: {
```

Comment:
Could you do this in build-systems.json? (same goes for other instances of adding build systems).

Reply:
Thanks, I'll update the pull request.

Reply:
I apologize for the mess, but I somehow made a new pull request instead of updating this one. So my changes are on #1144. I'll close this one out.